core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -312,7 +312,6 @@ private void apply(DeleteFileEntry entry) {
     Inode inode = mInodeStore.get(id).get();
 
     mInodeStore.remove(inode);
-    mInodeStore.removeChild(inode.getParentId(), inode.getName());
```

Comment:
Is this no longer necessary?

Reply:
Sorry, should have commented in the PR. The parent edge removal is moved to `RocksInodeStore`. It's already being done in `HeapInodeStore`, and specified to be done in the `InodeStore` interface.